app/views/admin/api_docs/base/_active_docs_table.slim

```diff
@@ -1,22 +1,26 @@
-table class="data"
-  thead
-    th Name
-    th System Name
-    th State
-    - if service.blank?
-      th API
-    th Swagger Version
-    th class='operations' = link_to('Create a new spec', new_api_docs_service_path(service), class: 'action add')
-  tbody
-    - for api_docs in api_docs_services
-      tr
-        td = link_to api_docs.name, preview_admin_api_docs_service_path(api_docs), title: 'Preview service spec'
-        td = api_docs.system_name
-        td = api_docs.published? ? 'visible' : 'hidden'
+div class="pf-c-card"
+  table class="data pf-c-table pf-m-grid-lg" role="grid" aria-label="Active docs table"
```

Comment:
It seems that you've removed `.data` everywhere, but not here.
1. is this intended?
2. is `.data` really useless? (we don't have any case of having multiple tables on the same page?)

Reply:
I must've have missed this. Besides `.data` being a really bad name for a selector, I replaced it in every JS script with an ID or pf-c-form. I think all `table.data`s were alone. In the few pages with multiple tables other classes are used.